spec/lib/nomis_client/moves_spec.rb

```diff
@@ -14,15 +14,15 @@
           from_location_nomis_agency_id: 'BXI',
           to_location_nomis_agency_id: 'BXI',
           date: '2019-08-19',
-          time_due: '17:00:00',
+          time_due: '2019-08-19T17:00:00',
           nomis_event_id: 468_536_961
         },
         {
           person_nomis_prison_number: 'G7157AB',
           from_location_nomis_agency_id: 'BXI',
           to_location_nomis_agency_id: 'WDGRCC',
           date: '2019-08-19',
-          time_due: '09:00:00',
+          time_due: '2019-08-19T09:00:00',
```

Comment:
We need to ensure that the `time_due` date value is the same as the `date`. Otherwise we will hit nasty daylight savings bugs (off by one hour errors).

It seems to me that the best way to avoid these is to store just one value that contains the date-time value for the move. However I suspect this won't always be practical - it's possible that the `time_due` could be on the next day?

So not sure what to suggest here. A starting point would be to check what data we are getting input via the API.